src/compiler/diagnosticMessages.json

```
@@ -4384,6 +4384,10 @@
         "category": "Error",
         "code": 6231
     },
+    "'{0}' is a 'get' accessor; did you mean to use it without '()'?": {

```

```suggestion
    "'{0}' is declared as a 'get' accessor and does not need to be called.": {
```

---

Actually, trying this out in an editor, I could see either one being good. It's up to you.

---

I could see the wording that mentions `()` being better if we could actually provide a code fix that deletes the (). I had some questions about that in my PR description, would you happen to have experience with code fixes and could help me figure out how to implement it cleanly?

---

With or without the codefix, I like the `()` message better since it shows what's wrong instead of naming it in terms that not everyone understands immediately.

---

re codefixes: There are some simple ones that you can basically copy, then you just have to learn how to extract the property access from the call, which should also be easy.

However, I'd rather just take the error alone and follow up with the codefix.

---

Actually, as a replacement for 'this expression is not callable', we should keep some elements of that:
```suggestion
    "This is not callable because it is a 'get' accessor; did you mean to use it without '()'?": {
```

(You already have an error span so I don't think you need to reproduce the expression in the error -- long expressions result in too-long errors.)